src/Nancy/DefaultRootPathProvider.cs

```diff
@@ -14,7 +14,7 @@ public class DefaultRootPathProvider : IRootPathProvider
         public string GetRootPath()
         {
 #if CORE
-            return Microsoft.Extensions.PlatformAbstractions.PlatformServices.Default.Application.ApplicationBasePath;
+            return System.IO.Directory.GetCurrentDirectory();

```

Comment:
Not entirely convinced that this is the best approach - the current working directory can change during the lifetime of an application, by such a simple thing as a call to ` Environment.CurrentDirectory = <some path>`